client/v3/doc.go

```diff
@@ -61,7 +61,7 @@
 //
 //  1. context error: canceled or deadline exceeded.
 //  2. gRPC error: e.g. when clock drifts in server-side before client's context deadline exceeded.
-//  See https://github.com/etcd-io/etcd/blob/main/api/v3rpc/rpctypes/error.go
+//     See https://github.com/etcd-io/etcd/blob/main/api/v3rpc/rpctypes/error.go
```

Comment:
This sentence might be started at a new line with a blank one above. Might be mis-formatted by the new Go tools.